main.go

```diff
@@ -1,106 +1,85 @@
 package main
 
 import (
-	"fmt"
 	"io/ioutil"
 	"os"
-	"path/filepath"
 	"strings"
 
 	"github.com/hashicorp/hcl2/hcl/hclsyntax"
 	"github.com/hashicorp/hcl2/hclparse"
 	"github.com/shoukoo/tf-verifier/walker"
-
+	flags "github.com/simonleung8/flags"
```

Comment:
What makes flags better than kingpin?

Reply:
Kingping gives much nicer documentation, but the actual feature of collecting together all the remaining file arguments is pretty awful. Eg:
To do it in kingping looks like this:

```type ipList []net.IP

func (i *ipList) Set(value string) error {
  if ip := net.ParseIP(value); ip == nil {
    return fmt.Errorf("'%s' is not an IP address", value)
  } else {
    *i = append(*i, ip)
    return nil
  }
}

func (i *ipList) String() string {
  return ""
}

func (i *ipList) IsCumulative() bool {
  return true
}

func IPList(s Settings) (target *[]net.IP) {
  target = new([]net.IP)
  s.SetValue((*ipList)(target))
  return
}
```

I don't really mind what CLI parsing tool is used, I wanted the usage of collecting all the files on the command line to work how other tools work. 
Looks like there's no shortage of tools.
https://awesome-go.com/#command-line